src/coord/src/coord.rs

```diff
@@ -3288,6 +3273,28 @@ impl Coordinator {
         }
     }
 
+    // Notify the timestamper thread that a source has been created or dropped.
+    async fn update_timestamper(&mut self, source_id: GlobalId, create: bool) {
```

Comment:
The if branches in this function don't share any code and each variant (create = true/false) is only called in one place. This suggests to me this shouldn't be a function? Light preference to just inlining the code.

Reply:
I think the create branch is called twice? (once when we create the source and once on restart). the drop variant is only called once though

Reply:
Oh right, I missed one. Not opposed to leaving it as is, just a thought I had.